### DISCUSSION TITLE
支付宝原生插件使用时如何正确传入onXXX事件？

### DISCUSSION THREAD

**Discussion by caidix** [NervJS/taro#15217] [Q & A]
在taro3.6.XX版本下使用支付宝原生插件，由于支付宝插件都是使用的onXXX 的方法。我按照官网使用bindonXXX 之类的各种方式，但是仍然无法正确调用。请问各位佬们知道我该怎么做吗？感激

**Comment by sangjq** [comment]
请问你解决这个问题了吗？我也遇到了，求指教

**Comment by chenweiliang6** [comment]
我也碰到了，找遍了没找到解决办法，这该咋办啊
